src/rules/prefer-lodash-method.js

```diff
@@ -42,6 +43,22 @@ module.exports = {
         const ignoredObjects = get(context, ['options', 0, 'ignoreObjects'], [])
         const usingLodash = new Set()
 
+        const nativeStringMap = {
+          endsWith: 'endsWith',
+          includes: 'includes',
+          padEnd: 'padEnd',
+          padStart: 'padStart',
+          repeat: 'repeat',
+          replace: 'replace',
+          split: 'split',
+          startsWith: 'startsWith',
+          toLowerCase: 'toLower',
+          toUpperCase: 'toUpper',
+          trim: 'trim'
+          //trimStart: 'trimStart', // es7 proposal
```

Comment:
I think you should remove the comments altogether until it's in the standard. Since it's only a proposal and it doesn't look like it's implemented anywhere yet ([Chrome and Firefox have `trimLeft` and `trimRight` but it's not standard](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/TrimLeft)), we'll readd it when necessary.